src/Draw/__snapshots__/DrawContainer.test.js.snap

```diff
@@ -164,5 +165,5 @@ exports[`<DrawContainer /> should render a basic prebuilt DrawContainer componen
               [36m</span>[39m
             [36m</div>[39m
             [36m<div[39m
-              [33mclass[39m=[32m\\"MuiInputBase-root MuiInput-root MuiI..."
+              [33mclass..."
```

Comment:
Seems like an unrelated change, worthy of a justification comment.

Reply:
snapshot updated from this: https://github.com/Bayer-Group/ol-kit/pull/309/files/99ad173cd0eafa255daa035de3bdc138b85dea74#diff-361b8ea1a952277c1efc12a7e05d9fae101b9480f46587dd4019ba19c03fc615R25

(but can be reverted)

Reply:
Perfect. Helps to point out this is a generated snap. Thanks.